web/src/components/home/subcomponents/upcoming_events/event_renderers/dyos_renderer.styles.js

```diff
@@ -0,0 +1,25 @@
+import { tableCellClasses } from "@mui/material";
+import theme from "../../../../../common/theme";
+
+const dyosRendererStyles = {
+  table: {
+    [`& .${tableCellClasses.root}`]: {
+      borderBottom: "none",
+    },
+  },
+  card: { width: { xs: "100%", sm: "450px" } },
+  cardContent: { padding: "24px 32px" },
+  eventTitle: { lineHeight: 1.5 },
+  eventSubtitle: { color: theme.palette.text.subtitle, lineHeight: 1.5 },
+  scheduleContainer: {
+    padding: { xs: "8px 0px 8px 4px", sm: "8px 0px 8px 16px" },
+  },
+  additionalDetails: {
+    color: theme.palette.text.secondaryBody,
+    paddingTop: "8px",
```

Comment:
We should use `rem` instead of `px` to make these styles responsive based on screen size

Reply:
Chatted offline - will prioritize getting a V1 out before adding all the bells and whistles. We should update it in the future